iree/compiler/Conversion/HLOToHLO/Passes.h

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//===- Passes.h - Codegen pass to convert from XLA-HLO to XLA-HLO ---------===//
+//
+// IREE specific passes used in the XLA-HLO to XLA-HLO. Some examples may be
```

Comment:
"used in the XLA-HLO to XLA-HLO." incomplete sentence

Reply:
Flushed out statement to "IREE specific passes used to sanitize XLA-HLO to IREE compatible XLA-HLO".